src/junit.rs

```diff
@@ -73,7 +83,9 @@ fn populate(
         failure_message: None,
         filename: rel_attrs.file,
         build_url: None,
-        computed_name: "".to_string(),
+        computed_name: ""
+            .try_into()
+            .context("Error converting computed name to ValidatedString")?,
```

Comment:
this is working on a literal, so `unwrap` should be fine, unless you still want to go the way of creating a constructor/`Default` for the empty string.